spec/facility_locations_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'spec_helper'
+
+RSpec.describe FacilityLocations do
+  
+  describe '#intialize' do
+    it 'can intialize' do
+      facility = FacilityLocations.new
+      expect(facility).to be_an_instance_of(FacilityLocations)
+    end
+  end
+
+  describe 'Creating Objects from Multiple Data Sets & Analyzing Data' do
+    it 'can #create_facilities from external data sources' do
+      colorado = DmvDataService.new.co_dmv_office_locations
+      co_offices = FacilityLocations.new
+      co_offices.create_facilities(colorado)
+
+      expect(co_offices.facilities.length).to be > 1
+      co_offices.facilities.each do |facility|
+        expect(facility).to be_an_instance_of(Facility)
+        expect(facility.name.nil?).to eq(false)
+        expect(facility.name).to be_an_instance_of(String)
+        expect(facility.address.nil?).to eq(false)
+        expect(facility.address).to be_an_instance_of(Array)
+        expect(facility.address[-2][:state]).to eq("CO")
+        expect(facility.phone.nil?).to eq(false)
+        expect(facility.phone).to be_an_instance_of(String)
+      end
```

Comment:
Nice job! Love that you used the enumerable in your test to test every element of the array.